SKIE/kotlin-compiler/core/src/commonMain/kotlin/co/touchlab/skie/phases/swift/CompileSwiftPhase.kt

```diff
@@ -48,7 +50,7 @@ class CompileSwiftPhase(
     }
 
     private fun createSwiftFileList(compilableFiles: List<SirCompilableFile>) {
-        val content = compilableFiles.joinToString("\n") { "'${it.absolutePath.absolutePathString()}'" }
+        val content = compilableFiles.joinToString("\n") { "'${it.relativePath.pathString}'" }
```

Comment:
Should this be always relative even with the `isRelativeSourcePathsInDebugSymbolsEnabled` being false?

Reply:
Yeah, because we're now always running the compilation from the `skieBuildDirectory.swift`, which results in the file name being relative, but `compilation dir` will be absolute. When `isRelativeSourcePathsInDebugSymbolsEnabled` is true, `compilation dir` gets set to `.`, making all the paths relative.